doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,64 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable
+================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`, it is possible to convert the app to an executable by taking a few precautions.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py
+
+Add the argument ``--onefile`` to put everything in a single file. Otherwise you will have a folder containing the application and dll dependencies.
```

Comment:
```suggestion
Add the argument ``--onefile`` to put everything in a single file. Otherwise you will have a directory containing the application and dynamic link library (DLL) dependencies.
```